deploy/secretmanage

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python2
+"""
+secretmanage -- manage production secrets!
+
+secretmanage allows you to upload new secrets. These secrets are then available
+in production (either in real production, staging or develop) by using the
+importsecret module.
+
+
+Usage:
+    secretmanage upload (--dev|--staging|--prod) <name> <secretstring>
```

Comment:
I'm not able to run `secretmanage` directly. I have to run it with the command `./secretmanage`. Is there anything I need to set up to run the command as described here?

Reply:
Hey! It's convention to leave the executable name like this, since it can be placed anywhere (`bin/`, `/usr/local/bin`, `/usr/bin/` etc) but you're right, it needs to be added with ./. How can we inform people of that?

Reply:
Just added the ./ to the command.